web/models.py

```
@@ -2103,6 +2096,13 @@
 class UserQuiz(models.Model):
     """Model for tracking user quiz attempts and responses"""
 
+    CORRECTION_STATUS = [
+        ("not_needed", "No Correction Needed"),
+        ("pending", "Correction Pending"),
+        ("in_progress", "Correction In Progress"),
+        ("completed", "Correction Completed"),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)